### ISSUE TITLE
Product images missing from confirmation emails

### ISSUE THREAD

**Issue by kieckhafer** [reactioncommerce/reaction#3391]
### Expected behavior
Product images should be displayed in order emails

### Actual behavior
A placeholder image is displayed in the an order email

### Steps to reproduce the behavior
- Upload an image to a product
- Create an order with that product
- See that the product image is not displaying in the order email received

### Versions
Node: 9.2.1
NPM: 5.6.0
Meteor Node: 8.8.1
Meteor NPM: 5.4.2
Reaction CLI: 0.24.2
Reaction: 1.6.0
Reaction branch: master
Docker: 17.11.0-ce

**Comment by zenweasel** [comment]
Can you give me some more specific steps to reproduce as I am seeing images in confirmation emails

**Comment by kieckhafer** [comment]
Just tested, I'm not seeing the image with just the most basic ordering of products.

- 
As an admin, I added an image to the product variant via the PDP page.
As a non-admin account, I ordered that product and received the email

<img width="780" alt="airmail" src="https://user-images.githubusercontent.com/4482263/33905937-2c4c30d6-df35-11e7-8237-aa98c4091013.png">


![basic_reaction_product_and_cart_completed](https://user-images.githubusercontent.com/4482263/33905923-2034f63e-df35-11e7-90f6-e12fb4780f06.png)

**Comment by codejockie** [comment]
@kieckhafer After so much testing I figured this is dependent on the mail app that was used to view the email.
On the browser I used Gmail and Yahoo but both didn't show the image. On MacOSX Mail app, the image is displayed.

**Comment by kieckhafer** [comment]
@codejockie My tests were all on a local Airmail app, which would have the images since it's running  locally. I think this is still an issue.

**Comment by impactmass** [comment]
Two things I've seen:

(1) There is a url malformation that happens here:
https://github.com/reactioncommerce/reaction/blob/941274223649fa795bfbb4ac6f392e28be998ee6/server/methods/core/orders.js#L701-L709

 A backslash gets stripped out, returning urls like "http:/example.com" instead of "http://example.com". _Some_ mail clients still resolve the url, some don't.

(2) The email template for the order confirmation contains:
```
{{#if variantImage}}
  <img src="{{variantImage}}"  width="50" height="50" alt="" />
{{else}}
  <img src="{{placeholderImage}}" width="50" height="50" alt="" />
{{/if}}
``` 
In some cases, the variantImage is not available, but a productImage is available, and can be used instead of falling to the default. This seems to be the case with @kieckhafer's test.

I'll create a PR for these.

**Comment by spencern** [comment]
Resolved in release [v1.6.4](https://github.com/reactioncommerce/reaction/releases/tag/v1.6.4)
